(DOCSP-17933): Developer mode permissions editing (#1304)
mongodb/docs-realm: source/sync/development-mode.txt

   you enable development mode, {+service-short+} does not enforce data access 
   rules. If your development database contains production data, developer
   mode would leave that data vulnerable to inadvertent or malicious access.
   
.. note:: Specify Data Access Rules with Development Mode Enabled

   By default, enabling development mode disables all data access rules,
   allowing any user to access any data. However, if your application requires
   permissions during schema development, you can specify data access

>    rules after enabling development mode.

Reviewer: How?

Author: I'll add a link to the instructions, since it works just as it does when not in developer mode. I originally thought I'd leave out the link to avoid cluttering the note, but on second thought I think it makes sense to clarify.

>    allowing any user to access any data. However, you can specify data access
>    rules after enabling development mode if your application requires limited
>    permissions even while developing your schema.

Author: ```suggestion
   allowing any user to access any data. However, if your application requires
   permissions during schema development, you can specify data access
   rules after enabling development mode.
```
